app/views/categories/index.html.erb

```diff
@@ -23,9 +27,8 @@
     </div>
 
     <div class="footer fixed-bottom py-4">
-        <% if user_signed_in? %>
-           <%= link_to "Logout", destroy_user_session_path, data: { turbo_method: :delete }, class: "btn main-color" %>
-        <% end %>
+    <h4 class="text-white"><%= link_to "Add a category", new_category_path, class: "text-whites" %></h4>
+
```

Comment:
[OPTIONAL]
- [ ] I recommend that you give this a little bit of styling to make it look like a button or actually change it to be a `button` element.

![image](https://user-images.githubusercontent.com/29071984/195862778-ee9b05dc-e3ae-4e7e-83a5-efb5dad0c28e.png)